README.md

```diff
@@ -30,19 +30,23 @@ Of course you can also use the provided classes within java as any other library
 
 ## Installing
 Download the repository from GitHub: `git clone https://github.com/acoli-repo/CoNLL-RDF.git`.
+Then use `git submodule update --init --recursive` to clone the fintan-core submodule.
 
-The source is compiled automatically once you run the tool via the `run.sh` bash script.
+The project is compiled automatically whenever you run the tool via the `./run.sh` bash script.  
+Experienced users may wish to modify the script to skip this for a small speed-up.
```

Comment:
Maven is a bit slow to start up. It's not an issue for running pipelines, I think, as the increase in time only scales linearly with number of calls to `./run.sh`, I think.